Main/Peripherals/src/Peripherals/ThrustController.cpp

```diff
@@ -53,12 +71,14 @@ std::pair<float,float> ThrustController::getReciprocalValues(float value) {
 }
 
 void ThrustController::setThrust(FloatPair forwardPair, FloatPair strafePair, float dive) {
-    leftForwardThruster_->Thrust(forwardPair.first);
-    rightForwardThruster_->Thrust(forwardPair.second);
+    leftForwardThruster_->Thrust(forwardPair.first * forwardTrim.first);
+    rightForwardThruster_->Thrust(forwardPair.second * forwardTrim.second);
+
     leftStrafeThruster_->Thrust(strafePair.first);
     rightStrafeThruster_->Thrust(strafePair.second);
-    forwardDiveThruster_->Thrust(dive);
-    rearDiveThruster_->Thrust(dive);
+
+    forwardDiveThruster_->Thrust(dive * diveTrim.first);
+    rearDiveThruster_->Thrust(dive * diveTrim.second);
```

Comment:
I think this should be adding the dive trim rather than multiplying it. But we'll have to check that next pool time and won't stop this pull request.